bootstrap/cmd/bootstrap/app/server.go

```diff
@@ -317,7 +323,7 @@ func Run(opt *options.ServerOption) error {
 		log.Fatalf("There was a problem loading the app: %v", err)
 	}
 
-	registryUri := fmt.Sprintf("github.com/kubeflow/kubeflow/tree/%v/kubeflow", opt.KfVersion)
+	registryUri := fmt.Sprintf("/opt/kubeflow/kubeflow")
```

Comment:
do we remove git support alltogether? I'm ok with that, just asking.

Reply:
Yes. I think we'll provide an image for each kubeflow version so user can choose what to use.